app/src/ui/history/commit-list-item.tsx

```diff
@@ -31,31 +23,20 @@ interface ICommitProps {
   readonly commit: Commit
   readonly selectedCommits: ReadonlyArray<Commit>
   readonly emoji: Map<string, string>
-  readonly isLocal: boolean
```

Comment:
Dude. I think this was the roughest one so far since the commit list items have so many things. :P